### PR TITLE
Testing the chat

### PR DESCRIPTION
**Pull request by tarelmadar**
Testing the chat using Selenium to control a Chrome web browser.

These tests will ensure that-

when a chat message is posted then it is seen by everyone in the same room
when a chat message is posted then it is not seen by anyone in a different room

### CODE REVIEW

File: sellotape/sellotape_dj/chat/tests.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+class ChatTests(ChannelsLiveServerTestCase):
+    serve_static = True  # emulate StaticLiveServerTestCase
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        try:
+            # NOTE: Requires "chromedriver" binary to be installed in $PATH
+            cls.driver = webdriver.Chrome("/Users/tarelgastonmadar/Downloads/chromedriver")
```
Comment (ItzhakiGuy) [inline sellotape/sellotape_dj/chat/tests.py]: Hi Tarel,
It seems you've committed a Selenium function to a directory at your own PC, if needed for tests, it should be a part of our project, and referenced respectively.
Does it needed?

**Comment by 11galpid** [comment]
Note, you fail the tests:

Run # Stop the build for any lint errors.
./sellotape/sellotape_dj/chat/tests.py:17:9: E722 do not use bare 'except'
        except:
        ^
1
Error: Process completed with exit code 1.

Please correct this error

**Review by 11galpid — CHANGES_REQUESTED** [review]
Changes need to be made to make the tests work

File: sellotape/sellotape_dj/chat/tests.py (hunk 1 above)
Comment (tarelmadar) [reply sellotape/sellotape_dj/chat/tests.py]: fixed this issue

**Review by 11galpid — APPROVED** [review]
Looks good

File: sellotape/sellotape_dj/chat/tests.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
+# chat/tests.py
+from channels.testing import ChannelsLiveServerTestCase
+from selenium import webdriver
```
Comment (ItzhakiGuy) [inline sellotape/sellotape_dj/chat/tests.py:3]: I have might miss something, I saw that the pipfile and it doesn't contain selenium, so it's not part of our virtual environment.
The testing of the chat app is used only by a user with selenium installed on his computer?
Would it be tested in the CI?
I may be mistaken so I would love to hear your elaborated answer ;-)

**Pull request closed without merging**